ants/registration/registration.py

```diff
@@ -1619,7 +1619,15 @@ def label_image_registration(fixed_label_images,
     type_of_deformable_transform : string
         Only works with deformable-only transforms, specifically the family
         of antsRegistrationSyN*[so] or antsRegistrationSyN*[bo] transforms.  
-        See 'type_of_transform' in ants.registration.
+        See 'type_of_transform' in ants.registration.  Additionally, one can
+        use a dictionary to pass a more tailored deformably-only transform 
```

Comment:
The usage says a dictionary here but it's actually a list. I think a dictionary would be better because it has named elements

Reply:
Thanks.  Changed that.

Reply:
> I think a dictionary would be better because it has named elements

Probably and I'm opening to changing this (as well as the multivariate extras option in ants.registration) depending on user feedback down the road.  I suspect the vast majority of users would simply use the family of `antsRegistrationSyN*` canned parameters and the users that want something more tailored probably won't get tripped up in the dictionary vs. list choice.